packages/core/src/features/cluster/kube-object-details/extension-api/reactively-hide-kube-object-detail-item.test.tsx

```diff
@@ -33,6 +41,8 @@ describe("reactively hide kube object detail item", () => {
       const store = {
         api,
         loadFromPath: async () => getKubeObjectStub("some-kind", "some-api-version"),
+        getByPath() {
```

Comment:
If I comment this line I'm getting very uninformative/misleading error description.. Can we improve that somehow?
Why there is nothing about that `store.getByPath(path: string)` not available?

<img width="1185" alt="Screenshot 2023-03-13 at 15 21 14" src="https://user-images.githubusercontent.com/6377066/224720028-19eb8918-ddbb-449f-ba7d-4299a3237d80.png">